apiserver/facades/controller/firewaller/firewaller.go

```diff
@@ -65,6 +65,11 @@ type FirewallerAPIV6 struct {
 	*FirewallerAPIV5
 }
 
+// FirewallerAPIV7 provides access to the Firewaller v7 API facade.
+type FirewallerAPIV7 struct {
+	*FirewallerAPIV6
```

Comment:
Shouldn't V6 have V7 and V7 have the main implementation?

Reply:
No as that is the pattern used in this facade. It doesn't jive with the other facade patterns but I opted for consistency in this one.